server/middleware/common-menu/testing-server.test.js

```diff
@@ -10,13 +11,14 @@ jest.mock('cryptr', () => {
 });
 
 describe('testing', () => {
-  const mockMongoClient = new mongo;
 
   beforeAll(async () => {
-    process.env.COMMON_MENU_COLLECTION='common-menu'
-    await mockMongoClient.startAndPopulateDB();
+    process.env.COMMON_MENU_COLLECTION='common-menu';
```

Comment:
let's save the old value in process.env and reassigned the old value in afterAll

check this stackoverflow article: https://stackoverflow.com/questions/48033841/test-process-env-with-jest